src/web/app/pages-admin/admin-search-page/admin-search-page.component.ts

```diff
@@ -26,6 +26,11 @@ export class AdminSearchPageComponent {
    */
   search(): void {
     this.accountService.searchAccounts(this.searchQuery).subscribe((resp: AdminSearchResult) => {
+      const hasStudents: boolean = !!(resp.students && resp.students.length);
+      const hasInstructors: boolean = !!(resp.instructors && resp.instructors.length);
+      if (!hasStudents && !hasInstructors) {
```

Comment:
Actually, you can make this an if-else block.
If there are no instructors and no students, then we wouldn't need to hide their links anyways

Reply:
Ok